mkdocs.yml

```diff
@@ -13,6 +13,7 @@ theme:
 
 markdown_extensions:
   - admonition
+  - attr_list
```

Comment:
Why `attr_list`? I was expecting `hal`, no?

Reply:
I have added this markdown extension in order to use attributes when working with pictures. For example, `{ align=center }` in the text.

Reply:
I have added a `hal.md` file and its menu entry (in `nav` in `mkdocs.yml`) in the previous pull request.